server/store/sqlstore/store.go

```diff
@@ -802,7 +821,7 @@ func (s *SQLStore) getActiveUsersCount(db sq.BaseRunner, dur time.Duration) (act
 
 //db:withReplica
 func (s *SQLStore) getConnectedUsers(db sq.BaseRunner, page, perPage int) ([]*storemodels.ConnectedUser, error) {
-	query := s.getQueryBuilder(db).Select("mmuserid, msteamsuserid, Users.FirstName, Users.LastName, Users.Email").From(usersTableName).LeftJoin("Users ON Users.Id = msteamssync_users.mmuserid").Where(sq.NotEq{"token": ""}).OrderBy("Users.FirstName").Offset(uint64(page * perPage)).Limit(uint64(perPage))
+	query := s.getQueryBuilder(db).Select("mmuserid, msteamsuserid, Users.FirstName, Users.LastName, Users.Email").From(usersTableName).LeftJoin("Users ON Users.Id = msteamssync_users.mmuserid").Where(sq.NotEq{"token": ""}).OrderBy("Users.FirstName").Offset(offset(page, perPage)).Limit(limit(perPage))
```

Comment:
Prior to this change, `gosec` was reporting:
> G115: integer overflow conversion int -> uint64

Open to suggestions on handling this more cleanly!